components/script/dom/readablestream.rs

```diff
@@ -282,24 +282,26 @@ impl ReadableStream {
 
     /// <https://streams.spec.whatwg.org/#readable-stream-error>
     pub fn error(&self, e: SafeHandleValue) {
-        // Set stream.[[state]] to "errored".
```

Comment:
I am using the correct steps numbers and aligning with rest of the code `step 1 == Set stream.[[state]] to "errored"`.

Reply:
I don't think there's an official guideline for spec comments within servo, but in my experience the step numbers can change significantly over time. Including spec text makes it easier to compare the current version of the specification to the one that was implemented (very useful when updating older code).

I'm not saying it's necessary for everyone to include the spec text in their comments, I'm just saying it would be nice to keep the comments that already exist.

Reply:
This was just placeholder and was misplaced, I added to correct step and correct step number

Reply:
I usually do something like `Step 2. Set stream.[[storedError]] to e.`

Reply:
Thanks for review, do you think it matter that much to add step and text?

Reply:
That's the new model that we're following in DOM these days.